AEPMessaging/Sources/MessagingPushTrackingStatus.swift

```diff
@@ -0,0 +1,47 @@
+/*
+ Copyright 2023 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import Foundation
+
+/// Represents a specific outcome resulting from tracking push notification interaction
+@objc(AEPMessagingPushTrackingStatus)
+public enum MessagingPushTrackingStatus: Int {
```

Comment:
i suppose we can drop `Messaging` from this name and just call it `PushTrackingStatus`?  having `Messaging` in the name feels a little redundant